jc/parsers/net_localgroup.py

```diff
@@ -0,0 +1,208 @@
+r"""jc - JSON Convert `net localgroup` command output parser
+
+
+Usage (cli):
+
+    $ net localgroup | jc --net-localgroup
+    $ net localgroup /domain | jc --net-localgroup
+    $ net localgroup Administrators | jc --net-localgroup
+    $ net localgroup Administrators /domain | jc --net-localgroup
+
+Usage (module):
+
+    import jc
+    result = jc.parse('net-localgroup', net_localgroup_command_output)
+
+Schema:
+
+    {
+        "account_origin":     string,
+        "domain":             string,
+        "comment":            string,
+        "groups": [
+            {
+                "name":       string
+                "members": [
+                              string
+                ]
+            }
+        ],
+    }
+
+
+    Notes:
+      [0] - 'lease_expires' and 'lease_obtained' are parsed to ISO8601 format date strings. if the value was unable 
+            to be parsed by datetime, the fields will be in their raw form
+      [1] - 'autoconfigured' under 'ipv4_address' is only providing indication if the ipv4 address was labeled as
+            "Autoconfiguration IPv4 Address" vs "IPv4 Address". It does not infer any information from other fields
+      [2] - Windows XP uses 'IP Address' instead of 'IPv4 Address'. Both values are parsed to the 'ipv4_address' 
+            object for consistency
+
+Examples:
+
+    $ net localgroup | jc --net-localgroup -p | jq
+    {
+        "account_origin": null,
+        "comment": null,
+        "domain": null,
+        "groups": [
+            {
+                "name": "Administrators",
+                "members": [
+                    "Administrator",
+                    "Operator",
+                    "ansible",
+                    "user1"
+                ]
+            }
+        ]
+    }
+    $ net localgroup Administrators | jc --net-localgroup -p | jq
+    $ net localgroup /domain | jc --net-localgroup -p | jq
```

Comment:
Looks like these examples are missing. no need to pipe through `jq` when using the `-p` option.